src/App.jsx

```diff
@@ -14,33 +15,112 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const arraysAreEqual = (arr1, arr2) => {
+  if (arr1.length !== arr2.length) {
+    return false;
+  }
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  for (let i = 0; i < arr1.length; i += 1) {
+    if (arr1[i] !== arr2[i]) {
+      return false;
+    }
+  }
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  return true;
+};
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+export const App = () => {
+  const [goods, setGoods] = useState(originalGoods);
```

Comment:
this state it's redundant

you can just create some functions that will return a sorted array
```
const goods = getSortedGoods(originalGoods, sortType, isReversed);
```